tests/mod.rs

```diff
@@ -0,0 +1,10 @@
+mod user_tests {
```

Comment:
I like this very much better than the approach I had previoulsy!

Reply:
New to me, but apparently its been here for a while :smile: looks like since Rust 2018 its sufficient to just declare the sub modules in the parent directory. We can actually rename it from mod.rs if we want, could just be tests.rs or something :man_shrugging: 

https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html